frontend/src/app/components/wp-calendar/wp-calendar.component.ts

```diff
@@ -159,6 +158,15 @@ export class WorkPackagesCalendarController implements OnInit, OnDestroy {
         workPackage: workPackage
       };
     });
+
+    // Instead of using two way bindings we manually trigger
+    // event rendering here. For whatever reasons, when embedded
+    // in a grid, having two way binding will lead to having constantly
+    // removed the events after showing them initially.
+    // It appears as if the two way binding is initialized twice if used.
+    setTimeout(() => {
```

Comment:
What do you need the timeout for?

Reply:
The fix is basically a rebuild of the two way binding (without the two way binding) for setting events as done in the [ng-fullcalendar npm package](https://github.com/Jamaks/ng-fullcalendar/blob/master/src/lib/src/calendar.component.ts#L26). The two way binding does not work as the assignment seems to be called twice (once with the desired results and once with undefined). 
Without the timeout
```
TypeError: Cannot read property 'renderEvents' of undefined
```
is thrown for whatever reasons.

Reply:
That's due to the calendar not yet being initialized. I allowed myself to push a commit that avoids using the timeout, and instead uses a callback from ng-fullcalendar.